packages/node-client/lib/index.ts

```diff
@@ -74,6 +77,82 @@ export class Nango {
         return response.data;
     }
 
+    public async proxy(config: ProxyConfiguration) {
+        validateProxyConfiguration(config);
+
+        const { providerConfigKey, connectionId, method: providedMethod } = config;
+
+        switch (providedMethod) {
+            case 'POST':
+            case 'post':
+                config.method = 'POST';
+                break;
+
+            case 'DELETE':
+            case 'delete':
+                config.method = 'DELETE';
+                break;
+
+            case 'PATCH':
+            case 'patch':
+                config.method = 'PATCH';
+                break;
+
+            case 'PUT':
+            case 'put':
+                config.method = 'PUT';
+                break;
+
+            default:
+                config.method = 'GET';
+                break;
+        }
+
+        const tokenResponse = await this.getToken(providerConfigKey, connectionId);
+
+        const url = `${this.serverUrl}/proxy`;
+
+        const headers = {
+            'Content-Type': 'application/json',
+            'Accept-Encoding': 'application/json'
+        };
+        const options = {
+            headers: this.enrichHeaders(headers)
+        };
+
+        config.token = tokenResponse;
+
+        return axios.post(url, config, options);
+    }
+
+    public async get(config: ProxyConfiguration) {
```

Comment:
These HTTP verbs are aliased for future ease of use for developers as we discussed @bastienbeurier